tests/dind.sh

```diff
@@ -6,23 +6,24 @@ K3D_IMAGE_TAG=$1
 # define E2E_KEEP to non-empty for keeping the e2e runner container after running the tests
 E2E_KEEP=${E2E_KEEP:-}
 
-####################################################################################
+# Max. time to wait for the runner container to be up
+RUNNER_START_TIMEOUT=${E2E_RUNNER_START_TIMEOUT:-10}
```

Comment:
maybe set this in the step to show it's possible to up the timeout without reading the script?